buddy/manifest.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "buddy",
+  "manifest_version": "1.0.0",
```

Comment:
Manifest Version is the version of the json. The version of the app should be a separate entry, `version`

Reply:
👍 That field is indeed the version of the manifest json and not the app version, so `manifest_version` is appropriate